README.md

```diff
@@ -38,90 +38,132 @@
+
+## Table of Contents
    * [Introduction](#introduction)
-   * [Guiding Principles](#guiding-principles)
    * [Getting Started](#getting-started)
+   * [Getting Help](#getting-help)
+   * [Example: GCN](#example-gcn)
+   * [Algorithms](#algorithms)
    * [Installation](#installation)
        * [Install StellarGraph using PyPI](#install-stellargraph-using-pypi)
        * [Install StellarGraph in Anaconda Python](#Install-stellargraph-in-anaconda-python)
        * [Install StellarGraph from Github source](#install-stellargraph-from-github-source)
-   * [Docker Image](#docker-image)
-   * [Running the examples](#running-the-examples)
-       * [Running the examples with docker](#Running-the-examples-with-docker)
-   * [Algorithms](#algorithms)
-   * [Getting Help](#getting-help)
-   * [Discourse Community](#discourse-community)
-   * [CI](#ci)
+       * [Docker Image](#docker-image)
    * [Citing](#citing)
    * [References](#references)
 
 ## Introduction
-**StellarGraph** is a Python library for machine learning on graph-structured (or equivalently, network-structured) data.
 
-Graph-structured data represent entities, e.g., people, as nodes (or equivalently, vertices),
-and relationships between entities, e.g., friendship, as links (or
-equivalently, edges). Nodes and links may have associated attributes such as age, income, and time when
-a friendship was established, etc. StellarGraph supports analysis of both homogeneous networks (with nodes and links of one type) and heterogeneous networks (with more than one type of nodes and/or links).
+The StellarGraph library offers state-of-the-art algorithms for [graph machine learning](https://medium.com/stellargraph/knowing-your-neighbours-machine-learning-on-graphs-9b7c3d0d5896), making it easy to discover patterns and answer questions about graph-structured data. It can solve many machine learning tasks:
 
-The StellarGraph library implements several state-of-the-art algorithms for applying machine learning methods to discover patterns and answer questions using graph-structured data.
-
-The StellarGraph library can be used to solve tasks using graph-structured data, such as:
 - Representation learning for nodes and edges, to be used for visualisation and various downstream machine learning tasks;
 - Classification and attribute inference of nodes or edges;
 - Link prediction;
-- Interpretation of node classification through calculated importances of edges and neighbour nodes for selected target nodes [8].
-
-We provide [examples](https://github.com/stellargraph/stellargraph/tree/master/demos/) of using `StellarGraph` to solve such tasks using several real-world datasets.
+- Interpretation of node classification through [calculated importances of edges and neighbour nodes for selected target nodes](https://medium.com/stellargraph/https-medium-com-stellargraph-saliency-maps-for-graph-machine-learning-5cca536974da) [8].
```

Comment:
This feels like too much jargon for an introduction?

Reply:
I'll remove everything from "through ..." onwards (but keep the link).